force-app/main/default/classes/HOT_DisableFreelancePart2Schedulable.cls

```diff
@@ -0,0 +1,64 @@
+public with sharing class HOT_DisableFreelancePart2Schedulable implements Database.Batchable<sObject>, Schedulable{
```

Comment:
Er det ikke mulig fra finish metoden i batch nr 1 og starte batch nr 2, så vi slipper å ha to Schedulable? Eller kanskje enda bedre fyra av en future-metode?